script/grant_revoke_gem_authority.rb

```diff
@@ -56,7 +54,6 @@
   'mkorcy' => "mkorcy@gmail.com",
   'ndushay' => "ndushay@stanford.edu",
   'tpendragon' => "tpendragon@princeton.edu",
-  'carrickr' => 'carrickr@umich.edu',
   'no_reply' => 'johnson.tom@gmail.com',
   'revgum' => 'revgum@gmail.com',
```

Comment:
I believe Josh Gum has left OSU.

Reply:
Josh is still at OSU but not at the Library so can be removed.

Reply:
Gracias!